src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import menuicon from '../../../assets/nav/menuIcon.png';
+import closeicon from '../../../assets/nav/closeIcon.png';
+import styles from "../../styles/Navbar.module.css";
+
+export const Navbar: React.FC= () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <nav className={styles.navbar}>
+      <a className={styles.title} href="/">

```

Comment:
Can we use `#` for href ? So it doesn't refresh the whole page.
```suggestion
      <a className={styles.title} href="#">

```

Reply:
fixed https://github.com/Sakshi-Simform/portfolio/pull/2/commits/387314e625ccf6bb575f42b84ba2cde20289fc21